### DISCUSSION TITLE
CHIME/FRB VOEvent Service will resume this week

### DISCUSSION THREAD

**Discussion by emilypetroff** [chime-frb-open-data/community#68] [General]
Hello,

The CHIME/FRB VOEvent service will resume this week. Due to a catastrophic failure of a hardware subsystem, the CHIME/FRB instrument has been offline since 16 September and has not been recording new events. CHIME/FRB systems were successfully restarted yesterday on 11 October and internal data validation checks are underway. Once we are confident that the system is operating nominally we will resume the CHIME/FRB VOEvent service. This will likely occur today (12 October) or tomorrow (13 October). If you require further information please feel free to get in touch with the CHIME/FRB team. 

We offer our sincere apologies for this outage and thank you for your patience.

Kind regards,
Emily Petroff
on behalf of CHIME/FRB

**Comment by antonioaddis** [comment]
Great news!
